src/libpmemobj/libpmemobj.vcxproj

```diff
@@ -111,7 +111,6 @@
     <ClInclude Include="container_ravl.h" />
     <ClInclude Include="container_seglists.h" />
     <ClInclude Include="memblock.h" />
-    <ClInclude Include="pvector.h" />
```

Comment:
This commit shouldn't exist. Each change should be squashed to the relevant commit.